prometheus-ksonnet/lib/prometheus-ha-mixin.libsonnet

```diff
@@ -0,0 +1,46 @@
+local container = $.core.v1.container;
+local statefulset = $.apps.v1.statefulSet;
+
+{
+  prometheus_config_0: $.prometheus_config,
+  prometheus_config_1: $.prometheus_config,
```

Comment:
It might be worth adding a cluster and replica label at this point, as per:

https://github.com/grafana/deployment_tools/blob/master/ksonnet/lib/default/prometheus.libsonnet#L59

https://github.com/grafana/deployment_tools/blob/master/ksonnet/lib/default/prometheus.libsonnet#L100